src/Integration.UnitTests/MefServices/VsSessionHostTests.cs

```diff
@@ -55,17 +55,7 @@ public void TestInitialize()
             this.stepRunner = new ConfigurableProgressStepRunner();
             this.configProvider = new ConfigurableConfigurationProvider();
 
-            var projectSystem = new ConfigurableVsProjectSystemHelper(this.serviceProvider);
```

Comment:
None of these are now required - some bits might have been left over from a previous incomplete refactoring.